.travis.yml

```diff
@@ -1,10 +1,34 @@
 language: python
-python:
-  - "2.7"
-  - "3.4"
-  - "3.5"
-  - "3.6"
+
+matrix:
+    include:
+        - python: 3.4
+          os: linux
+          dist: trusty
+          env: TOXENV=flake8
+        - python: 2.7
+          os: linux
+          dist: trusty
+          env: TOXENV=py34
```

Comment:
shouldn't that be TOXENV=py27 ?

Reply:
good spot. c&p error, fixed only one place...

i'll update the PR ...